picard/util/mbserver.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2021 Philipp Wolfer
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+from picard.config import get_config
+from picard.const import MUSICBRAINZ_SERVERS
+from picard.util import build_qurl
+
+
+def is_official_server(host):
+    """Returns True, if host is an official MusicBrainz server for the primary database.
+
+    Args:
+        host: the hostname
+
+    Returns: True, if host is an official MusicBrainz server, False otherwise
+    """
+    return host in MUSICBRAINZ_SERVERS
+
+
+def get_submission_host():
```

Comment:
Since it returns a tuple (host, port), we could rename it `get_submission_server()` and use an named tuple.

```python
ServerTuple = namedtuple('ServerTuple', ('host', 'port'))
```

```python
        return ServerTuple(MUSICBRAINZ_SERVERS[0], 443)
```

```python
server = get_submission_server()
do_something(server.host, server.port)
```

Reply:
I prepared a patch doing that: https://github.com/phw/picard/pull/10